README.md

```diff
@@ -165,7 +165,7 @@ As we can see, `myAnimal` object is an object of type `Cat`, not of type
 `Animal` That's because there is a difference between the type at compile-time
```

Comment:
Oops. I found another one. This might have been from me... but we're missing a period (.) after `Animal`